tests/cmux.py

```
@@ -150,21 +210,20 @@ def _default_socket_path() -> str:
 
-    # Otherwise, fall back to the newest discovered socket if there is one.
-    tagged = glob.glob("/tmp/cmux-debug-*.sock")
-    tagged.extend(glob.glob(os.path.join(_APP_SUPPORT_DIR, "cmux*.sock")))
-    tagged = [p for p in tagged if os.path.exists(p)]
-    if tagged:
-        tagged.sort(key=lambda p: os.path.getmtime(p), reverse=True)
-        for p in tagged:
-            if _can_connect(p, timeout=0.1, retries=2):
-                return p
+    if bundle_id.startswith(_DEFAULT_DEBUG_BUNDLE_ID):
+        tagged = glob.glob("/tmp/cmux-debug-*.sock")
```

Ruff (0.15.12)

[error] Probable insecure usage of temporary file or directory: "/tmp/cmux-debug-*.sock"

(S108)